create_jenkins_job.py

```diff
@@ -111,7 +110,7 @@ def main(argv=None):
         'linux-aarch64': {
             'label_expression': 'linux_aarch64',
             'shell_type': 'Shell',
-            'use_connext_default': 'false',
+            'ignore_rmw_default': data['ignore_rmw_default'] | {'rmw_connext_cpp', 'rmw_connext_dynamic_cpp'},
```

Comment:
Any RMW impl using Connext is being skipped on aarch64 atm.

Reply:
Could certainly be done. I just wanted to keep the changes minimal since the same was true before this patch. So I would rather keep that for a separate PR.

Reply:
Not sure I follow your reasoning. We've added a hard-coded list in multiple places in this PR whereas a flag was being set before. Given that we're modifying the rmw selection logic in this PR I don't see why we would save that change for a separate PR?

Reply:
Imo more variables implies more complexity. In order to understand a block of config values you will need to browse further up in the code to find what these variables are initialized with. Without the variables it is self contained (as before).

Please feel free to add the logic to this PR if you think it is important.